resources/migrations/001_update_migrations.yaml

```diff
@@ -6494,6 +6483,249 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  # This is necessary to make unique indexes using metabase_field.name case-sensitive for MySQL.
```

Comment:
As a summary, here is a list of the added migrations:
```
- Make metabase_field.name use case-sensitive collation for MySQL
- Add metabase_field.is_defective_duplicate
- Populate metabase_field.is_defective_duplicate
- Drop fk_field_parent_ref_field_id constraint with ON DELETE CASCADE
- Add fk_field_parent_ref_field_id constraint without ON DELETE CASCADE
- Remove idx_uniq_field_table_id_parent_id_name because it is redundant with idx_unique_field
- Remove the idx_uniq_field_table_id_parent_id_name_2col unique index because it blocks load-from-h2 with there
- Add unique_field_helper column to metabase_field
- Add unique constraint on metabase_field's (name, table_id, unique_field_helper)
```

Reply:
awesome summary, but I'd put this into the yaml file and add a bit of overview telling: migrations from `x` to `y` are used to fix defective fields issue.

Reply:
Sounds good, summary added in 749e003c5b88513533935632328fbef3fc93e3f4